src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -333,7 +333,7 @@ public function testNonDecoratedOutput()
             " 140/200 [===================>--------]  70%\n".
             " 160/200 [======================>-----]  80%\n".
             " 180/200 [=========================>--]  90%\n".
-            " 200/200 [============================] 100%",
+            ' 200/200 [============================] 100%',
```

Comment:
I won't let a bot tell me what's good :)

If we really want to make fabbot happy, we could change the previous lines to single quotes: `'...'."\n"`, but at this point, I don't really understand why we are spending hours on these CS "issues".